test/0-init-simple-json.sh

```diff
@@ -1,16 +1,32 @@
 #!/bin/bash
 
 packname="0-init-simple-pack"
+currentYear=$(date +"%Y")
 
 $DUB init -n $packname -f json
 
+cat > $packname/dub2.json <<- EOL
+{
+	"name": "$packname",
+	"authors": [
+		"$USER"
+	],
+	"dependencies": {},
+	"description": "A minimal D application.",
+	"copyright": "Copyright © $currentYear, $USER",
```

Comment:
I'm a bit surprised that this works, due to #715, I hope that's just the Travis setup. I don't know how difficult it is to get the users full name from the CLI, but we'd need either that or just searching for `"dependencies": {}` within the file would be a viable alternative.